sympy/physics/vector/frame.py

```diff
@@ -930,7 +930,7 @@ def orient_space_fixed(self, parent, angles, rotation_order):
 
         >>> from sympy import symbols
         >>> from sympy.physics.vector import ReferenceFrame
-        >>> q1, q2, q3 = symbols('q1, q2, q3')
+        >>> q1, q2, q3 = symbols('q1, q2, q3', real=True)
```

Comment:
https://github.com/sympy/sympy/pull/23171 may have some bearing on this, so it needs to be merged first and this PR retested.